src/components/navigation.js

```diff
@@ -12,6 +13,7 @@ const Navigation = () => (
           <Link style={{ textDecoration: 'none', color: 'black' }} to="categories">CATEGORIES</Link>
         </li>
       </ul>
+      <span><IoIosPerson className="profile" /></span>
     </nav>
   </div>
```

Comment:
- [ ] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
![Screenshot from 2023-04-06 19-33-41](https://user-images.githubusercontent.com/83042742/230512670-c3db75ae-8f98-4a5f-86c0-51c1edb045ab.png)

### Your Design
![Screenshot from 2023-04-06 19-34-00](https://user-images.githubusercontent.com/83042742/230512717-6f39abba-ccb6-434f-aab9-61455279e276.png)